README.md

```diff
@@ -33,6 +33,12 @@ composer install
 composer test-coverage
 ```
 
+To run tests in a docker container (without coverage):
```

Comment:
I do not mind one way or the other, if you desire I shall remove it: however I thought it could help next person who would like to contribute to this repository. I found it easier to test the different versions of php with docker using this command instead of having to manually install them on the host machine.